tests/pkcli/build1_data/1.out/srv/host/v3.radia.run/etc/nginx/conf.d/v3.radia.run.conf

```diff
@@ -0,0 +1,43 @@
+server {
+    listen v3.radia.run:80;
+    server_name v3.radia.run;
+    return 301 https://v3.radia.run$request_uri;
+}
+
+server {
+    listen v3.radia.run:443 ssl;
+    server_name v3.radia.run;
+    root /usr/share/nginx/html;
+    ssl_certificate /etc/nginx/conf.d/v3.radia.run.crt;
+    ssl_certificate_key /etc/nginx/conf.d/v3.radia.run.key;
+    keepalive_timeout 0;
+
+    location / {
+        root /home/vagrant/src/radiasoft/rsconf/tests/pkcli/build1_work/1/srv;
+        try_files $uri $uri/index.sh $uri/index.html =404;
+    }
+
+    location = /host/ {
+        return 404;
+    }
+
+    location ~ ^/host/([^/]+)/ {
+        auth_basic "*";
+        auth_basic_user_file /etc/nginx/conf.d/rsconf_auth;
+        # DEBUG: return 200 $remote_user;
+        if ($remote_user != $1) {
+            return 403;
+        }
+        root /home/vagrant/src/radiasoft/rsconf/tests/pkcli/build1_work/1/srv;
+    }
+
+    location /kickstart/ {
+        allow v9.radia.run;
```

Comment:
This list will get really long with the number of hosts we have. Much better to use a CIDR.